src/pages/api/dataSources/keystone.ts

```diff
@@ -1,21 +1,25 @@
-import { RESTDataSource } from 'apollo-datasource-rest'
+import { RESTDataSource } from '@apollo/datasource-rest'
+import type { KeyValueCache } from '@apollo/utils.keyvaluecache'
 
 class KeystoneAPI extends RESTDataSource {
-  constructor() {
-    super()
-    this.baseURL = process.env.KEYSTONE_URL
+  override baseURL = process.env.KEYSTONE_URL
+
+  constructor(options: { cache: KeyValueCache }) {
```

Comment:
The migration guide has the cache included now for data sources, although I'm not sure if we need it yet.